documentation/docs/guidedtour.md

```diff
@@ -0,0 +1,200 @@
+# Getting Started with Project "Piper"
+
+Follow this guided tour to become familiar with the basics of using project "Piper".
+
+The public sample application [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] will be enriched with a pipeline which syncs the sources, builds these as MTA and deploys the result into a Cloud Foundry environment. The application contains a simple `nodejs` application. Deployed as web service, it serves static data.
+
+**Recommendation:** We recommend to clone the sample application [cloud-cf-helloworld-nodejs][cloud-cf-helloworld-nodejs] and execute the instructions on your own repository. See **(Optional) Sample Application**.
+
+The stated instructions assume the use of this application.
+
+## Prerequisites
+
+* You have installed a Linux system with at least 4 GB memory. **Note:** We have tested our samples on Ubuntu 16.04. On Microsoft Windows, you might face some issues.
+* You have installed the newest version of Docker. See [Docker Community Edition](https://docs.docker.com/install/). **Note:** we have tested on Docker 18.09.6.
+* You have installed Jenkins 2.60.3 or higher. **Recommendation:** We recommend to use the `cx-server` toolkit. See **(Optional) Install the `cx-server` Toolkit for Jenkins**. **Note:** If you use your **own Jenkins installation** you need to care for "Piper" specific configuration. Follow [my own Jenkins installation][guidedtour-my-own-jenkins].
+* Your system has access to [GitHub.com][github].
+
+## (Optional) Install the `cx-server` Toolkit for Jenkins
+
+`cx-server`is a lifecycle management toolkit that provides Docker images with a preconfigured Jenkins and a Nexus-based cache to facilitate the configuration and usage of Jenkins.
+
+To use the toolkit, get the `cx-server` script and its configuration file `server.cfg` by using the following command:
+
+```sh
+docker run -it --rm -u $(id -u):$(id -g) -v "${PWD}":/cx-server/mount/ ppiper/cx-server-companion:latest init-cx-server
+```
+
+When the files are downloaded into the current directory, launch the Jenkins server by using the following command:
+
+```sh
+./cx-server start
+```
+
+For more information on the Jenkins lifecycle management and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server][devops-docker-images-cxs-guide].
+
+Please also consult the [S/4HANA Cloud SDK Blog Series][sap-blog-s4-sdk-first-steps] and in particular the post on setting up [Continuous Delivery for S/4HANA extensions][sap-blog-ci-cd].
+
+## (Optional) Sample Application
+
+Copy the sources of the application into your own Git repository. While we will ask you to fork the application's repository into a public GitHub space, you can use any version control system based on Git. **Note:** A public GitHub repository is visible to the public. The configuration files may contain data you don't want to expose.
```

Comment:
> [public fork on github] ... The configuration files may contain data you don't want to expose.

I personally think this is too weak. Basically we should clearly recommend to fork into a private repo. If a user does not follow this suggestion - OK. But that is a difference to what we have now.